app/components/markdown-block.js

```diff
@@ -90,6 +115,9 @@ const useStylesFromThemeFunction = createUseStyles(theme => ({
       fontSize: '1rem',
       textAlign: 'left',
     },
+    '& p:first-of-type':{
+      marginTop: props.iconName === '' ? '16px' : '0px', //removes margin-top from first paragraph as per figma design in issue 43
```

Comment:
For responsiveness, please use rem instead of px.   Please see this: https://github.com/EnCiv/.github/wiki/React-Coding-and-Style-Guidelines for why. 

Also for the 32px below.